tsconfig.json

```diff
@@ -11,7 +11,8 @@
     "noFallthroughCasesInSwitch": true,
     "resolveJsonModule": true,
     "sourceMap": true,
-    "outDir": "build"
+    "outDir": "build",
+    "skipLibCheck": true
```

Comment:
Why is this needed?

Reply:
We have errors coming from type definitions for newly-introduced packages:
```sh
node_modules/@types/passport-strategy/node_modules/@types/passport/index.d.ts:23:13 - error TS2717: Subsequent property declarations must have the same type.  Property 'authInfo' must be of type 'any', but here has type 'AuthInfo | undefined'.

23             authInfo?: AuthInfo | undefined;
               ~~~~~~~~

  node_modules/@types/passport/index.d.ts:14:13
    14             authInfo?: any;
                   ~~~~~~~~
    'authInfo' was also declared here.

node_modules/@types/passport-strategy/node_modules/@types/passport/index.d.ts:24:13 - error TS2717: Subsequent property declarations must have the same type.  Property 'user' must be of type 'any', but here has type 'User | undefined'.

24             user?: User | undefined;
               ~~~~

  node_modules/@types/passport/index.d.ts:15:13
    15             user?: any;
                   ~~~~
    'user' was also declared here.
```

I think this is because we build in strict mode, but maybe such definitions aren't done fort